### DISCUSSION TITLE
Cut color sequence pictures

### DISCUSSION THREAD

**Discussion by Liudeke** [mlavik1/UnityVolumeRendering#243] [Q&A]
Hello author, I wonder if this function can be realized by obtaining coronal and sagittal plane images through cross-sectional sequence color pictures, as shown in the following picture
![QQ截图20240419094332](https://github.com/mlavik1/UnityVolumeRendering/assets/30310819/b16a3383-7749-4c4d-9af9-11d6915a3f98)

**Comment by mlavik1** [comment]
Hi! I'm sorry for the late reply. These weeks have been a bit busy.

If I understand you correctly, you have a 3D dataset containing colour values (not only density values) - is that right?
There is currently no support for such datasets, however it should be possible to implement.
You'd need to change the "data" array of VolumeDataset to be an array of colours, change the generated texture and also update the shader (wouldn't use a  tranfer function then, I guess).

Are you working on some open datasets by the way?

**Reply by Liudeke** [reply]
no, this is not open data
